tests/test_bolt2-30-channel_type-open-accept-tlvs.py

```diff
@@ -166,15 +165,12 @@ def test_open_channel_bad_type(runner: Runner, with_proposal: Any) -> None:
         Block(blockheight=102, txs=[tx_spendable]),
         Connect(connprivkey="02"),
         ExpectMsg("init"),
-        TryAll(
-            # BOLT-a12da24dd0102c170365124782b46d9710950ac1 #9:
-            # | 20/21 | `option_anchor_outputs`          | Anchor outputs
-            Msg("init", globalfeatures="", features=bitfield(12, 21)),
-            # BOLT #9:
-            # | 12/13 | `option_static_remotekey`        | Static key for remote output
-            Msg("init", globalfeatures="", features=bitfield(12)),
-            # And not.
-            Msg("init", globalfeatures="", features=""),
+        # BOLT #9:
+        # | 12/13 | `option_static_remotekey`        | Static key for remote output
+        Msg(
+            "init",
+            globalfeatures=runner.runner_features(globals=True),
+            features=runner.runner_features(additional_features=[13]),
```

Comment:
and here

Reply:
This is a typo as well. I will go through the entire pr to see if there are any more typos and fix them.